core-data/src/commonMain/sqldelight/io/github/droidkaigi/confsched2022/data/TimetableItemSessionSpeakers.sq

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE timetableItemSessionSpeakers (
```

Comment:
Can we have better names for the intermediate tables?🤔

Reply:
How about "TimetableItemSpeakerCrossRef" from
https://developer.android.com/training/data-storage/room/relationships#many-to-many